tests/Feature/Dashboard/TeamDownlineTest.php

```
@@ -0,0 +1,512 @@
+            'name' => $name,
+            'email' => "user{$userId}@example.test",
+        ]);
+
+        DB::table('lead_agents')->insert([
+            'id' => $id,
+            'company_id' => $this->companyId,
+            'user_id' => $userId,
+            'parent_agent_id' => $parentId,
+        ]);
+
+        return LeadAgent::find($id);
```

PHPStan (2.2.8)

[error] Method Tests\Feature\Dashboard\TeamDownlineTest::agent() should return App\Models\LeadAgent but returns array<Eloquent>|Illuminate\Database\Eloquent\Collection|Illuminate\Database\Eloquent\Model|null.


(return.type)